### PR TITLE
add task sidebar and i18n

### PR DESCRIPTION
**Pull request by giacomos**

### CODE REVIEW

File: src/ploneintranet/workspace/browser/tiles/sidebar.py [hunk 1]
```diff
@@ -253,6 +271,25 @@ def children(self):
                 'dpi': dpi})
         return items
 
+    def tasks(self):
+        items = []
+        catalog = getToolByName(self.context, name="portal_catalog")
```
Comment (adamcheasley) [inline src/ploneintranet/workspace/browser/tiles/sidebar.py]: If you use the api to get the portal_catalog, as you have done on L166 we can remove the import of getToolByName.

File: src/ploneintranet/workspace/browser/tiles/sidebar.py [hunk 2]
```diff
@@ -12,7 +12,10 @@
 from DateTime import DateTime
 from Products.CMFPlone.utils import safe_unicode
 from Products.CMFCore.utils import _checkPermission as checkPermission
+from Products.CMFCore.utils import getToolByName
+from ploneintranet.todo.behaviors import ITodo
```
Comment (adamcheasley) [inline src/ploneintranet/workspace/browser/tiles/sidebar.py]: It doesn't look like pi.todo is a dependancy. Am I missing something here?

File: src/ploneintranet/workspace/browser/tiles/sidebar.py [hunk 3]
```diff
@@ -156,30 +159,45 @@ def __call__(self):
...
+                        todo.status = u'done'
+                    else:
+                        todo.status = u'tbd'
                 IStatusMessage(self.request) \
-                    .addStatusMessage('Calendar visibility changed', 'success')
+                    .addStatusMessage(_(u'Changes applied'), 'success')
+            else:
+                if form.get('title') and form.get('title') != ws.title:
+                    ws.title = form.get('title').strip()
+                    IStatusMessage(self.request) \
```
Comment (adamcheasley) [inline src/ploneintranet/workspace/browser/tiles/sidebar.py]: These should be changed to use `api.portal.show_message`

File: src/ploneintranet/workspace/browser/tiles/sidebar.py [hunk 4]
```diff
@@ -253,6 +271,25 @@ def children(self):
...
+        brains = catalog(path=current_path, portal_type=ptype)
+        for brain in brains:
+            obj = brain.getObject()
+            todo = ITodo(obj)
+            data = {
+                'id': brain.UID,
+                'title': brain.Description,
+                'content': brain.Title,
+                'url': brain.getURL(),
+                'checked': todo.status == u'done' and True or False
```
Comment (adamcheasley) [inline src/ploneintranet/workspace/browser/tiles/sidebar.py]: `and True or False` is superfluous here.

**Comment by adamcheasley** [comment]
There is two merge commits within this pull request, which is a bit noisy. Rebasing may have solved that.

**Comment by mattss** [comment]
@pilz There are some valid code review issues raised here.

**Comment by mattss** [comment]
@pilz Did you mean to merge this yet?

**Comment by pilz** [comment]
Just merged it to the next branch level for integration. We don't merge the final pull request to master before all comments are resolved

**Comment by mattss** [comment]
OK cool thanks.

**Pull request merged**